### PR TITLE
Feature/metobs parameters

### LINKED ISSUE #91: 'Datum' is not returned for parameter no 2
It seems MetObs is having trouble parsing data returned for parameter 2 (and possibly other?). We should explore why and adapt the code to allow reading all signals.

### PR DESCRIPTION
**Pull request by mgcth**
## Description

Add support for all parameters for Metobs. Close #91.

### CODE REVIEW

**Comment by mgcth** [comment]
@docNord, could you add integration tests. Don't need one for each parameter, just the different types.

We could probably clean up the code a bit by storing the parameter type in the data class.

**Comment by docNord** [comment]
Alright, I will have a look and continue working in this PR. You're sure the format types are set according to your implementation in constants @mgcth ?

File: src/smhi/metobs.py [hunk 1]
```diff
@@ -603,11 +605,18 @@ def _parse_data(self, raw_data: str) -> None:
...
+        columns = self.data.columns
+
+        if any([c for c in METOBS_PARAMETER_TIM if c in columns]):
+            datetime_columns = METOBS_PARAMETER_TIM
+        elif any([c for c in METOBS_PARAMETER_DYGN if c in columns]):
+            datetime_columns = METOBS_PARAMETER_DYGN
+        elif any([c for c in METOBS_PARAMETER_MANAD if c in columns]):
+            datetime_columns = METOBS_PARAMETER_MANAD
+        else:
+            raise TypeError("Can't parse type.")
```
Comment (mgcth) [inline src/smhi/metobs.py:617]: @docNord should be possible to be cast as a dict. Want to give it a try? Otherwise, I'm done with this as is for now.

Comment (docNord) [reply src/smhi/metobs.py:617]: I wouldn't mind giving it a go tomorrow - I pulled the code yesterday and started looking, but I will gladly discard any changes I might have put in and look again tomorrow after the meeting.

Comment (mgcth) [reply src/smhi/metobs.py:617]: Let's keep it as is.

**Pull request merged**